### PR TITLE
[WIP] create multiple authors API endpoint from given csv file

### PR DESCRIPTION
**Pull request by reeshipaul**

### CODE REVIEW

File: cmd/beast/create_admin_author.go [hunk 1]
```diff
@@ -86,3 +94,139 @@ var createAdminCmd = &cobra.Command{
...
+			fmt.Printf("Csv File path not provided")
+			os.Exit(1)
+		}
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		csvFile := filepath.Join(BEAST_GLOBAL_DIR, CsvFile)
+
+		config.InitConfig()
+
+		auth.Init(core.ITERATIONS, core.HASH_LENGTH, core.TIMEPERIOD, core.ISSUER, config.Cfg.JWTSecret, []string{core.USER_ROLES["author"]}, []string{core.USER_ROLES["admin"]}, []string{core.USER_ROLES["contestant"]})
```
Comment (cypherean) [inline cmd/beast/create_admin_author.go:113]: What does this do?

Comment (cypherean) [inline cmd/beast/create_admin_author.go:113]: What does this do?

**Comment by cypherean** [comment]
Resolve conflicts and add a check that the commands are run only if the user who's running it is an admin

**Comment by cypherean** [comment]
Resolve conflicts and add a check that the commands are run only if the user who's running it is an admin

File: cmd/beast/create_admin_author.go (hunk 1 above)
Comment (reeshi-18) [reply cmd/beast/create_admin_author.go:113]: InitConfig loads the config from the global config file
auth.Init initializes the constants in pkg/auth.go from contants.go

**Pull request closed without merging**